nix/checks.nix

```diff
@@ -9,16 +9,44 @@
   }: {
     checks =
       {
+        black =
+          pkgs.runCommandLocal "black" {
+            nativeBuildInputs = [pkgs.python3.pkgs.black];
+          } ''
+            cd ${../.}
+            for py in $(find . -name "*.py" ! -path "*venv*" ! -path "*eggs*"); do
```

Comment:
All the eggs and venvs are gone, we can probably drop this (and the `make clean` target) in a followup.

Reply:
Marked as resolved for now, let's leave this to a followup.